internal/debug/debugger.go

```diff
@@ -0,0 +1,106 @@
+package debug
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const PROMPT string = "dbg> "
```

Comment:
The `string` type is not required here. Also, do we want to export that constant?